src/lib/WebhookLogger.js

```diff
@@ -1,15 +1,15 @@
 // @flow
 
 import type { IWebhookLogger } from '../types';
-
-import logger from './logger';
+import Logger from './logger';
+const logger = Logger.createModuleLogger(module);
 
 class WebhookLogger implements IWebhookLogger {
   _lastLog: Array<any>;
 
-  log(...args: Array<any>) {
-    this._lastLog = args;
-    logger.log(...args);
+  log(...argsarray: Array<any>) {
+    this._lastLog = argsarray;
+    logger.info({ args: argsarray }, 'WebHook');
```

Comment:
I dont event know when this is called. The text is original from previous code. 
I renamed to "WebHookLogger called"